src/runAll.js

```diff
@@ -62,51 +62,56 @@ module.exports = function runAll(config) {
       renderer
     }
 
-    if (tasks.length) {
-      // Do not terminate main Listr process on SIGINT
-      process.on('SIGINT', () => {})
+    // If all of the configured "linters" should be skipped
+    // avoid executing any lint-staged logic
+    if (tasks.every(task => task.skip())) {
+      console.log(
+        'No linters executed. No staged files match any of provided globs in "lint-staged.linters".'
+      )
+      return 'No tasks to run.'
```

Comment:
I’m wondering if we’re using this return value anywhere and if we should use `return` for the other case.

Reply:
Yeah, I was uncertain about this so I left it as is.
Let me know what you want to do. I feel it's best to probably just leave it unless you think it won't break anything. (Also is this function exported outside of the library? If so maybe someone has code relying on that string return!)